backend/app/compose.yaml

```diff
@@ -3,23 +3,21 @@ version: '3.7'
 volumes:
   postgres:
   pgadmin:
-  keycloak_data:
-    driver: local
 networks:
   common-network:
     driver: bridge
 services:
   postgres:
     container_name: phlat_postgres
-    image: 'postgres:latest'
+    image: 'postgres:13.8'
```

Comment:
aligning this local postgres version with the one we have in cloud.

Reply:
Postgres was updated to v13.13 on AWS but that's ok. It seems that we will be required to update the version in local env as required. Minor version difference should not affect us in any way.

Reply:
changing it to 13.13

Reply:
done